utility/common_utils.js

```diff
@@ -431,7 +432,11 @@ function stringifyProps(propReaderObject, comments) {
 }
 
 function getHomeDir() {
-	let homeDir = undefined;
+	// for hermetic tests
+	if (process.env.OVERRIDE_HOME_DIR) {
```

Comment:
Does this function differently than `process.env.ROOTPATH`?

Reply:
> Does this function differently than `process.env.ROOTPATH`?

Yes, but that seems to work just as well in its place, so I'll issue a followup PR that switches over. Thanks!